packages/utils/navigation.ts

```diff
@@ -107,6 +112,11 @@ const navConfig: {
     Governance: "governance",
     UserPublicProfile: "user/:id/:tab?",
     RiotersFooter: "rioters-footer",
+    AdministrationDashboard: "launchpad/admin",
+    LaunchpadApplications: "launchpad/applications",
+    ApplicationReview: "launchpad/application-review",
+    ReadyLaunchpadApplications: "launchpad/ready-applications",
+    AllProjectAdministrationDash: "launchpad/all-applications",
```

Comment:
I think `launchpad/all-projects` is better no ? This is the title in the UI screen, and with that we can't have a confusion with `applications`